### ISSUE TITLE
`ml overview` gets unfinished capture

### ISSUE THREAD

**Issue by rungitta** [TACC/Lmod#538]
I just installed the latest version of `Lmod` and was going to try `overview`. It gave me this output:
```
$ ml --version

Modules based on Lua: Version 8.5.21  2021-10-22 09:54 -05:00
    by Robert McLay mclay@tacc.utexas.edu

$ ml overview
/bin/lua: /opt/software/lmod/lmod/libexec/Master.lua:1027: unfinished capture
stack traceback:
        [C]: in function 'string.find'
        /opt/software/lmod/lmod/libexec/Master.lua:1027: in function 'Master.overview'
        /opt/software/lmod/lmod/libexec/cmdfuncs.lua:144: in function 'Overview'
        /opt/software/lmod/lmod/libexec/lmod:516: in function 'main'
        /opt/software/lmod/lmod/libexec/lmod:576: in main chunk
        [C]: in ?
```

Any ideas what causes this?

**Comment by rtmclay** [comment]
Probably it is an alias that Lmod is not handling correctly.  Please send me the output of ```ml -t av``` or put it in a git gist or similar so that I can see what the problem is.

**Comment by rungitta** [comment]
Here you go:
https://gist.github.com/rungitta/1028b48f4fe0e45f05d06c422fca3525

**Comment by rtmclay** [comment]
What version of Lua are you using?

**Comment by rungitta** [comment]
5.3.4 (rpm install)

-- 
Axel Rosén, Head Engineer
Research Infrastructure Services Group, USIT
University of Oslo, Norway

**Comment by rtmclay** [comment]
Please change line 1027 in Master.lua to:
```
         if (entry:find("%(@")) then
```

to see if the '%' fixes the issue.

**Comment by rungitta** [comment]
That did they trick! Thank you very much :)

Is this a bug or is something wrong with my setup?

**Comment by rungitta** [comment]
No worries! :)

**Comment by rtmclay** [comment]
I have found a test that exposed this bug and I have released version 8.5.22.  Thanks again for the bug report!

**Comment by rtmclay** [comment]
Closing this issue.  If there are any problem please feel free to re-open this issue.
